apps/web/components/ui/ImagePreview/ImagePreview.vue

```diff
@@ -8,32 +8,33 @@
         <strong>{{ props.name }}</strong>
         <div v-if="meta.width && meta.height" class="text-xs mt-1">{{ meta.width }} x {{ meta.height }} px</div>
       </div>
-      <button class="text-gray-500 hover:text-gray-700 text-xl leading-none" @click="close">&times;</button>
+      <SfTooltip label="Click here to upload" placement="top" :show-arrow="true" class="z-10">
+        <button
+          type="button"
+          aria-label="Close preview"
```

Comment:
```suggestion
          aria-label="Close preview and open upload"
```
Should match the full functionality